liver_imaging_analysis/engine/engine.py

```diff
@@ -103,7 +116,8 @@ def get_network(self, network_name, **kwargs):
                 parameters of network, if exist.
         """
         networks = {
-            "monai_2DUNet" : monai.networks.nets.UNet,
+            "monai_2DUNet": monai.networks.nets.UNet,
+            "monai_ResUNet": monai.networks.nets.SegResNet,
```

Comment:
This might be misleading, ResUNet usually refers to Unet with residuals. Even though they are similar, there are still some differences.

Reply:
done